src/components/CredentialCard/CredentialCard.tsx

```diff
@@ -220,6 +259,13 @@ export function CredentialCard({
           </dl>
         </section>
       )}
+
+      {deleting && (
+        <ConfirmDelete
```

Comment:
Do we benefit from having this as a component instead of placing the code inline?

Reply:
Good point. I guess not